docs/pages/docs.md

```diff
@@ -5,23 +5,69 @@ layout: default
+
+[docstrings]: https://peps.python.org/pep-0257/#what-is-a-docstring
+[GitHub pages]: https://docs.github.com/en/pages
+[our template]: https://github.com/UCL-ARC/python-tooling?tab=readme-ov-file#using-this-template
+[template-docs-dot-yaml]: https://github.com/UCL-ARC/python-tooling/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/docs.yml
+
+## Documentation build tools
+
+| Name      | Short description                                                                                                                                               | 🚦  |
+| --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
+| [MkDocs]  | Generates documentation from Markdown files, with a plugin to generate API documentation. A good plugin ecosystem and balance of usability and customisability. | 🟢  |
+| [Sphinx]  | Generates documentation from reStructuredText or Markdown files, long been the de-facto standard and very widely used. Mature plugin ecosystem.                 | 🟠  |
+| [gitbook] | General documentation builder; integrates with GitHub.                                                                                                          | 🟠  |
+| [pdoc]    | Auto-generates API documentation from docstrings, beginner friendly but with less of a plugin ecosystem than others.                                            | 🟠  |
+
+<details markdown="block">
+<summary>More details about Sphinx</summary>
+
+We marginally recommend [MkDocs] over [Sphinx] due to it's ease of use,
+preference for Markdown, and more support for a variety of docstring styles.
+
+However the [Sphinx] tool has long been the de-facto standard in the scientific Python ecosystem. It is widely
+used, customisable, and well tested. If you need a [Sphinx
+extension](#sphinx-extensions) that does not have an equivalent [MkDocs
+plugin](https://github.com/mkdocs/catalog), or if you are part of a community
+that heavily uses [Sphinx] then we recommend you use that
+instead.
+
+### See also
+
+- Our internal discussions about which to recommend ([#16](https://github.com/UCL-ARC/python-tooling/issues/16) and [#187](https://github.com/UCL-ARC/python-tooling/issues/187)).
+- [An interesting related discussion](https://github.com/encode/httpx/discussions/1220).
+
 </details>
 
+<!-- URLS used above -->
+
+[MkDocs]: https://www.mkdocs.org/
+[Sphinx]: https://www.sphinx-doc.org/en/master/
+[gitbook]: https://www.gitbook.com/
+[pdoc]: https://pdoc.dev/
+
+## MkDocs plugins
+
+| name                                                          | short description                            | 🚦  |
+| ------------------------------------------------------------- | -------------------------------------------- | :-: |
+| [mkdocstrings-python](https://mkdocstrings.github.io/python/) | Automatically generates api reference pages. | 🟢  |
```

Comment:
```suggestion
| [mkdocstrings-python](https://mkdocstrings.github.io/python/) | Automatically generates API reference pages. | 🟢  |
```

Reply:
Thanks. This is me typing too fast, I think.